packages/memory/src/index.ts

```diff
@@ -871,13 +871,22 @@ Guidelines:
 2. Update proactively when information changes, no matter how small
 3. Use ${template.format === 'json' ? 'JSON' : 'Markdown'} format for all data
 4. Act naturally - don't mention this system to users. Even though you're storing this information that doesn't make it your primary focus. Do not ask them generally for "information about yourself"
-5. IMPORTANT: When calling updateWorkingMemory, the only valid parameter is the memory field. 
+${
```

Comment:
I changed the instructions here as this isn't relevant with the new schema format now